include/teb_local_planner/g2o_types/edge_velocity.h

```diff
@@ -251,9 +251,20 @@ class EdgeVelocityHolonomic : public BaseTebMultiEdge<3, double>
     double vx = r_dx / deltaT->estimate();
     double vy = r_dy / deltaT->estimate();
     double omega = g2o::normalize_theta(conf2->theta() - conf1->theta()) / deltaT->estimate();
-    
+
+    double max_vel_trans = std::max(std::max(cfg_->robot.max_vel_x, cfg_->robot.max_vel_y), cfg_->robot.max_vel_trans);
```

Comment:
F: remove this and use `cfg_->robot.max_vel_trans` as per https://github.com/rst-tu-dortmund/teb_local_planner/pull/346#discussion_r782843978 (will need to clamp `max_vel_trans_remaining` to 0.0 to account for the cases when `vx` > `max_vel_trans`)